Makefile.work

```diff
@@ -118,7 +118,8 @@ SONIC_BUILD_INSTRUCTION :=  make \
                            KERNEL_PROCURE_METHOD=$(KERNEL_PROCURE_METHOD) \
                            HTTP_PROXY=$(http_proxy) \
                            HTTPS_PROXY=$(https_proxy) \
-                           SONIC_ENABLE_SYSTEM_TELEMETRY=$(ENABLE_SYSTEM_TELEMETRY)
+                           SONIC_ENABLE_SYSTEM_TELEMETRY=$(ENABLE_SYSTEM_TELEMETRY) \
+                           $(SONIC_OVERRIDE_BUILD_VARS)
```

Comment:
is this used?

Reply:
I added SONIC_OVERRIDE_BUILD_VARS to have ability to pass variables to slave makefile, specifically to pass MLNX_SDK_SOURCE_BASE_URL in build time this way, instead of introducing vendor build parameters in root makefile

usage:
```
make SONIC_BUILD_JOBS=12 SONIC_OVERRIDE_BUILD_VARS="MLNX_SDK_SOURCE_BASE_URL=http://arc-build-server/stepanb/sx_sdk_eth/sx_sdk_eth-4.3.0136/SOURCES MLNX_SDK_VERSION=4.3.0136" target/sonic-mellanox.bin
```